docs/development.md

```diff
@@ -51,6 +51,12 @@ To stop the dev server, press `Ctrl + C`.
 
 Back-end dev server is the Jetty-based server handling all the business logic, including data storage.
 
+### Pre-requisites
+
+In order for the back-end to properly work, you need to have a running database instance and a full-text search service instance (if you are supporting one). The instances can either be a local emulator or a running production instance.
+
```

Comment:
I think it might cause confusion here when it says "you need to have a running database instance and a full-text search service instance" but in `search.md` it is mentioned that setup of search service is optional. Do you think so?

Reply:
`(if you are supporting one)`

Reply:
It sounds abit less direct compared to `... and optionally a full-text search service instance.`, but I guess it should be ok since we reiterated it in `search.md`.